### ISSUE TITLE
Can't delete forward

### ISSUE THREAD

**Issue by Jean-MarcS** [postfixadmin/postfixadmin#371]
Hi.

First, thanks for the great work and making postfix that easy to admin.

I can't (or haven't found how) remove a forward on a mailbox.

When I click on "alias", I can see the forward destination adress, but if I remove it and validate, I've got the pEdit_alias_goto_text_error1

I created it with the CLI, and will delete it with the CLI as well. But it was one of the administrator who wanted to do that, and they don't have access to CLI.

Is there a way to do this operation, have I missed it ?

Or is it a missing feature :-) ?

Regards
Jean-Marc S.

**Comment by DavidGoodwin** [comment]
try changing / checking these in your config settings : 


```
// If you don't want edit alias tab (user mode) set this to 'NO';
$CONF['edit_alias'] = 'YES';

// Alias control for superadmins
$CONF['alias_control'] = 'YES';

```

**Comment by Jean-MarcS** [comment]
Hi David, thank you for answering !

It's all YES in the actual conf.

I found a way to put it back the way I wanted it, but it's not very clean.

First I deleted the entry in the `alias` database.

On the web side, the account was still marked as "Forward", and (of course) there were nothing left in the alias form.

So I created back the entry in the DB again, and looked at how it was done on other accounts.

Then I changed the destination of the alias

It was 

```address@domain1.com => address@domain2.com```

and I changed it to

```address@domain1.com => address@domain1.com```

And now it's marked as "Mailbox" and not "Forward" anymore.
